Dockerfile

```diff
@@ -1,6 +1,6 @@
 # A image for building paddle binaries
 # Use cuda devel base image for both cpu and gpu environment
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvcr.io/nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
```

Comment:
This Dockerfile will be deprecated later, try consider build using images under https://github.com/PaddlePaddle/buildtools

Reply:
As a seperate issue, this base image is for building docker image to release on NGC(nvidia gpu cloud). For general user and dockerhub images, it is better to use dockerhub cuda image.
We may need some discussion on where to put the dockerfile that build the NGC image.